bot/log.py

```diff
@@ -14,11 +14,27 @@
 TRACE_LEVEL = 5
 
 
+class CustomLogger(Logger):
+    """Custom implementation of the `Logger` class with an added `trace` method."""
+
+    def trace(self, msg: str, *args, **kwargs) -> None:
+        """
+        Log 'msg % args' with severity 'TRACE'.
+
+        To pass exception information, use the keyword argument exc_info with
+        a true value, e.g.
+
+        logger.trace("Houston, we have an %s", "interesting problem", exc_info=1)
+        """
+        if self.isEnabledFor(TRACE_LEVEL):
+            self._log(TRACE_LEVEL, msg, args, **kwargs)
```

Comment:
It's better to use the non-internal `.log()` method to do the work for us, it essentially does [what you're doing here but extra](https://github.com/python/cpython/blob/778b07565e38cc94aa90318eb47b9cd09716756a/Lib/logging/__init__.py#L1536).

Reply:
So to clarify, this would become `self.log(...)`?

Reply:
Yep, and you pass `*args` instead of `args`

Reply:
Committed as part of [ca2d4a5](https://github.com/python-discord/bot/pull/1831/commits/1adae82ebc32fc39636f4ca2d20ec2e2b3c3fbd0)